src/encoded/static/vis_defs/PCS_vis_def.json

```diff
@@ -0,0 +1,51 @@
+{
+    "PCS": {
+        "rule": "'{assay_term_name}' == 'pooled clone sequencing'",
+        "assay_composite": {
+            "longLabel":  "Collection of ENCODE pooled clone sequencing experiments",
+            "shortLabel": "ENCODE pooled clone sequencing"
+        },
+        "longLabel":  "{assay_title} of {replicates.library.biosample.summary} - {accession}",
+        "shortLabel": "{assay_title} of {biosample_term_name} {accession}",
+        "sortOrder": [ "Biosample", "Replicates", "Views" ],
+        "Views":  {
+            "tag": "view",
+            "group_order": [ "Signal of unique reads" ],
+            "groups": {
+                "Signal of unique reads": {
+                    "tag": "aSIG",
+                    "visibility": "full",
+                    "type": "bigWig",
+                    "viewLimits": "0:10",
+                    "autoScale": "off",
```

Comment:
Not sure how important this is. I did noticed we typically turn off autoScale and set a limit of 0:10. But I also noticed on the demo the signal could be off chart if we do that. Again, more like a confirmation: we indeed want to do that?

Reply:
Good catch. I could just set autoScale to be on and take away the viewLimits to make sure the signal is viewed ok. Probably the signal of unique reads in the STARR file should be changed as well. I guess we usually don't turn on autoScale to avoid the track being distorted by outliers? But it seems like for the FCC signal of unique read/bigWig files this isn't the issue.